dependencies.gradle

```diff
@@ -3,12 +3,13 @@
  * SPDX-License-Identifier: GPL-3.0-only
  */
 ext.versions = [
-    minSdk: 21,
-    targetSdk: 29,
-    compileSdk: 29,
+    minSdk: 23,
```

Comment:
minSdk 23 :thinking:

Reply:
I'm moving all my projects to minSdk 23, API 21/22 have broken drawable behaviour, missing crypto and other problems that are not worth our time. It'll also allow us to fix #526 for all users in a consistent fashion.

Reply:
Oh okay :+1: